infra/aws/terraform/prow-build-cluster/resources/monitoring/grafana/datasources.yaml

```diff
@@ -0,0 +1,15 @@
+# TODO: Update with local prometheus
```

Comment:
Is this a leftover? This file seems to already use a local Prometheus setup.

Reply:
Oops, my bad.